tests/unittest_inference.py

```diff
@@ -5316,6 +5316,11 @@ def test_slice_inference_in_for_loops_not_working() -> None:
         assert inferred == util.Uninferable
 
 
+def test_slice_zero_step_does_not_raise_ValueError() -> None:
+    node = extract_node("x = [][::0]; x")
+    assert next(node.infer()) == util.Uninferable
```

Comment:
Please add a test for the other failure too.
Either via `infer` or a direct call to `Const.getitem`.